<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="re_pass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>username </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>re-type password </string>
   </property>
  </widget>
  <widget class="QComboBox" name="month">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="day_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="year">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>date of birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>month</string>
   </property>
  </widget>
  <widget class="QLabel" name="dytd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>day</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>gender</string>
   </property>
   <widget class="QRadioButton" name="male">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="account">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>account type</string>
   </property>
   <widget class="QRadioButton" name="user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>favorite genre(s)</string>
   </property>
   <widget class="QCheckBox" name="action">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="comedy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="romance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="drama">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horror">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="other">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="register_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_error">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*not matching</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_error">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*age lesss than 12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="register_error">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*all fields mjust be filled </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_error">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*already existing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
